Tests/Orso.Arpa.Api.Tests/IntegrationTests/ProjectsControllerTests.cs

```diff
@@ -117,7 +118,7 @@ public async Task Should_Get_Participations_By_Id()
             result.Should().BeEquivalentTo(expectedResult, opt => opt.WithStrictOrderingFor(dto => dto.Id));
         }
 
-        [Test, Order(100)]
+        [Test, Order(10000)]
```

Comment:
10000 haben wir für delete-tests reserviert (und existiert auch so, ganz unten)

Reply:
Tests mussten in der Reihenfolge getauscht werden, weil es sonst zu Fehlern kam.
Kann im Nachgang nochmal angeschaut werden